src/test/scala/org/constellation/TransactionProcessorTest.scala

```diff
@@ -31,7 +31,7 @@ class EdgeProcessorTest extends FlatSpec {
   dbActor.setAutoPilot(new TestActor.AutoPilot {
     def run(sender: ActorRef, msg: Any): TestActor.AutoPilot = msg match {
       case DBGet(`srcHash`) =>
-        sender ! Some(AddressCacheData(100000000000000000L, None))
+        sender ! Some(AddressCacheData(100000000000000000L, 100000000000000000L, None))
```

Comment:
I'll try to understand the PR a bit, contentwise, in the evening. 

At this quoted line 34 I suppose you could use a `testAmount = 100000000000000000L` handle and, while at it, define it as some power of 10.
I see not reason to short `balance` or `amount` to three or four letter words. Also, GitHub complaints about files without missing newlines at the bottom.

Reply:
@buckysballs consider it moral support :P 
Thought worth going through it when it's out as an PR, but I'm still trying to desgin a code review sort of video and roughly only made my way up to the primitives for now.